pkg/operation/common/types.go

```diff
@@ -221,12 +221,6 @@ const (
 	// KubeSchedulerServerName is the name of the kube-scheduler server.
 	KubeSchedulerServerName = "kube-scheduler-server"
 
-	// CalicoKubeControllersDeploymentName is the name of calico-kube-controllers deployment.
```

Comment:
Let's keep these deployment/daemonset names. We need them for the health check. Yes, it's not perfect because we cannot assume that calico is deployed and if any other extension is used the health check will always fail. However, we don't have a mechanism yet to allow extensions to report health status. Also, there is no other network extension yet. So, let's keep it for now until we can work on this problem (only after the API migration topic).